### PR TITLE
Add .NET Core support, strong name to assemblie

### PR DESCRIPTION
**Pull request by qmfrederik**
Based on @smbecker's work, this PR adds support for .NET Core to Portable.Licensing
Additionally, it also strongs name the assemblies.

### CODE REVIEW

**Comment by RichiCoder1** [comment]
It seems unlikely the @dnauck will be merging this or updating Portable.Licensing. If that ends up the case, maybe this could be pushed all on it's own as a fork?

**Comment by qmfrederik** [comment]
I've (temporarly) published my version of Portable.Licensing on NuGet as [CoreCompat.Portable.Licensing](https://www.nuget.org/packages/CoreCompat.Portable.Licensing).

If @dnauck merges this PR, I'll happily pull the package from NuGet (no need to needlessly create forks) but until then, feel free to use that package as a stop-gap solution.

File: src/Portable.Licensing.Tests/Portable.Licensing.Tests.csproj [hunk 1]
```diff
@@ -1,76 +1,21 @@
...
-  -->
-  <Import Project="..\..\.paket\paket.targets" />
+
   <ItemGroup>
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\packages\NUnit\lib\nunit.framework.dll</HintPath>
-      <Private>True</Private>
-      <Paket>True</Paket>
-    </Reference>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (RichiCoder1) [inline src/Portable.Licensing.Tests/Portable.Licensing.Tests.csproj:18]: What's this about?

Comment (qmfrederik) [reply src/Portable.Licensing.Tests/Portable.Licensing.Tests.csproj:18]: This is auto-generated by VS can can be safely ignored. That said, I'm doing cleanup on my fork of Portable.Licensing that probably exceeds the scope of this PR - so do my work on a separate branch & bring this PR back into its previous shape.

**Pull request closed without merging**